api/apps/api/src/modules/access-control/access-control.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+
+import { AccessControlService } from './access-control.service';
+import { ProjectAclModule } from '@marxan-api/modules/projects-acl';
+
+@Module({
+  imports: [ProjectAclModule],
+  providers: [AccessControlService],
+  exports: [AccessControlService],
+})
+export class AccessControlModule {}
```

Comment:
Think of:

```
// keep abstract class
abstract class ProjectAccessControl {}

// implementation
export class ProjectAclService extends ProjectAccessControl { }

// inject
@Module({
  providers: [
    {
         provide: ProjectAccessControl,
         useClass: ProjectAclService
    }
  ],
  exports: [ProjectAccessControl]
})
```

Thanks to that we may be able to get rid of "dummy" facade.

Also "types" of response would be still within module contract instead of particular implementation.

Reply:
sounds like a better approach, thanks!

Reply:
@kgajowy https://github.com/Vizzuality/marxan-cloud/pull/634/commits/5dd9ac0e3a3d4a482286538125e23839ea9725a2